recipes/fastqvalidator/gcc_zlib.patch

```diff
@@ -0,0 +1,33 @@
+--- bam/SamInterface.cpp	2012-10-24 15:53:29.000000000 -0500
```

Comment:
You can but comments at the top of a patch file. Explaining the reason for this patch will help down the line.